.github/workflows/gsas2pkg.yml

```diff
@@ -6,23 +6,24 @@
 # Then add the key as a Repository secret named ANACONDA_TOKEN using
 #   the Settings/Secrets.../Actions 
 
-# TODO: could we build the Raspberry Pi-64 with Conda convert? (N.B. must use pip
-#    for wxpython rather than conda)
+# TODO: could we build the Raspberry Pi-64 with Conda convert? (N.B. 
+#    must use pip install for wxpython rather than conda)
 #
 name: build gsas2pkg
 
-#on: [push, pull_request, workflow_dispatch]   # why build on pull request?
-#on: [push, workflow_dispatch]
-on: [workflow_dispatch]
-# run on all platforms this if any of the following change: 
-#    install/gitstrap.py
-#    install/g2pkg/meta.yaml
-# for mac/linux:
-#    install/g2pkg/build.sh
-#    install/g2pkg/post-link.sh
-# for windows:
-#    install/g2pkg/post-link.bat
-#    install/g2pkg/bld.bat
+on:
+  workflow_dispatch:   # nice for debugging
+  push:
+    branches: [ "master" ]
+    paths:                  # want this to run only when files in
+                            # g2pkg or gitstrap.py changes
+      - install/gitstrap.py    
+      - install/g2pkg/*
+  pull_request:             # When is this triggered? Do I need this? 
+    branches: [ "master" ]
+    paths:
+      - install/gitstrap.py    
+      - install/g2pkg/*
```

Comment:
When a pull request is opened from some branch into "master" ("master" does not exist in this repository the default branch is called "main") AND when the diff between "master" and the pull request branch contains changes to any files matching the paths listed. This action will run again every time the aforementioned pull request is updated with new commits.